src/types/jsonforms.ts

```diff
@@ -0,0 +1,18 @@
+// THIS IS NOT AN EXHAUSTIVE LIST
```

Comment:
Threw this together mainly just for the multiline secret. These values are important for our app and JSONForms... so not storing them in the component that uses it.